DigitalLearningSolutions.Web/Helpers/RegistrationEmailValidator.cs

```diff
@@ -0,0 +1,104 @@
+﻿namespace DigitalLearningSolutions.Web.Helpers
+{
+    using System;
+    using DigitalLearningSolutions.Data.DataServices;
+    using DigitalLearningSolutions.Data.Services;
+    using DigitalLearningSolutions.Web.Extensions;
+    using DigitalLearningSolutions.Web.ViewModels.Register;
+    using Microsoft.AspNetCore.Mvc.ModelBinding;
+
+    public static class RegistrationEmailValidator
+    {
+        private const string WrongEmailForCentreErrorMessage =
+            "This email address does not match the one held by the centre";

```

Comment:
I think we could tweak this to get some more information across, e.g. "Either your primary email or centre email must match the one held by the centre." What do you think?

Reply:
I'm in favor of that